classes/class.ilAdobeConnectXMLAPI.php

```diff
@@ -247,6 +249,45 @@ public function logout($session)
 			return false;
 		}
 	}
+
+	/**
+	 * @return null|string
+	 */
+	public function getApiVersion()
+	{
+		global $DIC;
+		$ilLog = $DIC->logger()->root();
+
+		$url = $this->getApiUrl(array('action' => 'common-info'));
+
+		$context    = array(
+			'http'  => array('timeout' => 4),
+			'https' => array('timeout' => 4)
+		);
+		$ctx        = $this->proxy($context);
+		$xml_string = file_get_contents($url, false, $ctx);
+		$xml        = simplexml_load_string($xml_string);
+
+		if($xml && $xml->common->cookie != "")
```

Comment:
Is this (`$xml->common->cooki`) really required?

Reply:
no ;-)

Reply:
fixed